torchaudio/models/wavernn.py

```diff
@@ -35,6 +36,13 @@
 }
 
 
+def bits_to_normalized_waveform(label: torch.Tensor, bits: int) -> torch.Tensor:
```

Comment:
Should this be public?

Reply:
No, it shouldn't. I've changed it to a private function [here](https://github.com/pytorch/audio/pull/1704/files#diff-7f4ad259da182d3840fe4c75dd0c0b7b9fe30f31503df0eff87b1e5c9378cac4R39-R43).